packages/@react-spectrum/breadcrumbs/docs/Breadcrumbs.mdx

```diff
@@ -46,32 +45,14 @@ category: Navigation
 ```
 
 ## Content
-Breadcrumbs follow the [Collection Components](../react-stately/collections.html) API, accepting only static children. Similar to the Menu, Breadcrumbs accept an `<Item>` component, each with a `key` prop which is passed to the `onAction` handler to identify the selected item. Basic usage of Breadcrumbs, seen in the example above, shows a list of <TypeLink links={collectionsDocs.links} type={collectionsDocs.exports.Item} /> components, populated with a string. See [Events](#events) for more information.
+Breadcrumbs follow the [Collection Components](../react-stately/collections.html) API, accepting only static children.
```

Comment:
Shouldn't there be something about the last item reflecting the current page and not being interactive.

Reply:
I feel like that is more in the realm of design and is covered by the spectrum docs